core/src/main/scala/org/apache/spark/SerializableWritable.scala

```diff
@@ -41,7 +41,7 @@ class SerializableWritable[T <: Writable](@transient var t: T) extends Serializa
   private def readObject(in: ObjectInputStream): Unit = Utils.tryOrIOException {
     in.defaultReadObject()
     val ow = new ObjectWritable()
-    ow.setConf(new Configuration())
+    ow.setConf(new Configuration(false))
```

Comment:
To clarify, this is `@DeveloperAPI` and has been marked as such for a bunch of releases, which is why we don't want to rename / move it.